rerun_py/docs/gen_common_index.py

```diff
@@ -108,6 +108,7 @@ class Section:
     Section(
         title="Timeline functions",
         func_list=[
+            "set_index",
             "set_time_sequence",
             "set_time_seconds",
             "set_time_nanos",
```

Comment:
Shouldn't these be removed from the documentation?

Reply:
I did, but then I got confusing errors 😞